src/test/java/de/adorsys/keycloak/config/test/util/KeycloakVersion.java

```diff
@@ -0,0 +1,36 @@
+/*-
+ * ---license-start
+ * keycloak-config-cli
+ * ---
+ * Copyright (C) 2017 - 2020 adorsys GmbH & Co. KG @ https://adorsys.de
+ * ---
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ---license-end
+ */
+
+package de.adorsys.keycloak.config.test.util;
+
+public class KeycloakVersion {
+
+    private static final String KEYCLOAK_VERSION_ENV_NAME = "KEYCLOAK_VERSION";
+
+    public static boolean isKeycloak8() {
+        String version = getKeycloakVersion();
+        return version != null && version.startsWith("8.");
+    }
+
+    private static String getKeycloakVersion() {
+        return System.getenv(KEYCLOAK_VERSION_ENV_NAME);
```

Comment:
Could you use `System.getProperty("keycloak.version");` instead `System.getenv` for consistency?

Like: 
https://github.com/adorsys/keycloak-config-cli/blob/f7ed19ab9a8b7e4f2191317cba436b63f20802f7/src/test/java/de/adorsys/keycloak/config/ImportExportedRealmIT.java#L45

It uses the value from maven directly, e.g. this is not covering `mvn clean verify -Dkeycloak.version=8.0.2`